SlideMenuControllerSwift/String.swift

```diff
@@ -14,10 +14,11 @@ extension String {
     }
     
     func substring(_ from: Int) -> String {
-        return self.substring(from: self.characters.index(self.startIndex, offsetBy: from))
+        let index = self.index(self.startIndex, offsetBy: from)
+        return String(self.suffix(from: index))
```

Comment:
just info that I found,
you can use  `self[index...]`  instead of suffix and will give you the same